packages/@aws-cdk/aws-appmesh/test/test.virtual-node.ts

```diff
@@ -113,19 +273,15 @@ export = {
 
     const node = mesh.addVirtualNode('test-node', {
       dnsHostName: 'test.domain.local',
-      listener: {},
```

Comment:
I have removed the empty listener as we will not have a default listener for a node. It would be valid to have a node without listener and then add it later.

[Reference](https://github.com/aws/aws-cdk/pull/10793#discussion_r515393247)